patches/api/0377-Add-option-to-hide-entities-by-default.-This-option-.patch

```diff
@@ -0,0 +1,36 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: CrystallDEV <m.hasselder@gmx.de>
+Date: Sun, 27 Mar 2022 15:05:07 +0200
+Subject: [PATCH] Add option to hide entities by default. This option is
+ currently very strict and overrides all other changes to visibility and
+ interaction. Fixes #7395
+
+
+diff --git a/src/main/java/org/bukkit/entity/Entity.java b/src/main/java/org/bukkit/entity/Entity.java
+index 8bc6876c82935988436597161fa0ec94c032174b..6e7e34af7870c5c31157c7963fe6283d0a8a861d 100644
+--- a/src/main/java/org/bukkit/entity/Entity.java
++++ b/src/main/java/org/bukkit/entity/Entity.java
+@@ -730,6 +730,23 @@ public interface Entity extends Metadatable, CommandSender, Nameable, Persistent
+     @Nullable
+     Location getOrigin();
+ 
++    /**
++     * Gets if the entity is hidden by default
++     *
++     * @return True if the entity is hidden by default
++     */
++    boolean isHiddenByDefault();
++
++    /**
++     * Hides this entity from all players by default
++     */
++    void hideEntityByDefault();
++
++    /**
++     * Shows this entity to all players by default
++     */
++    void showEntityByDefault();
```

Comment:
This should be one setter, `setHiddenByDefault`.

Reply:
Indeed, missed updating these in the API when changing the base functions